100Change2020/ansible/roles/100Change2020/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+# Because the overrides MUST come after the originals in LocalSettings, we
```

Comment:
Because no one should be editing the local settings file, and the comment talks about ansible things (deleting the block), the comment should more appropriately here because the decisions are all made here.  I did update it to make t more clear.